src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -14,11 +15,38 @@ import { startPlayingAndWaitForVideo } from '../../web_platform/util.js';
 const kHeight = 16;
 const kWidth = 16;
 const kFormat = 'rgba8unorm';
-const kVideoSources: string[] = [
-  'red-green.webmvp8.webm',
-  'red-green.bt601.vp9.webm',
-  'red-green.mp4',
-  'red-green.theora.ogv',
+
+interface VideoExpectations {
+  video: string;
+  redExpectation: Uint8Array;
+  greenExpectation: Uint8Array;
+}
+const kVideoExpectations: VideoExpectations[] = [
+  {
+    video: 'red-green.webmvp8.webm',
+    redExpectation: new Uint8Array([0xd9, 0x00, 0x00, 0xff]),
+    greenExpectation: new Uint8Array([0x01, 0xef, 0x00, 0xff]),
+  },
+  {
+    video: 'red-green.theora.ogv',
+    redExpectation: new Uint8Array([0xd9, 0x00, 0x00, 0xff]),
+    greenExpectation: new Uint8Array([0x01, 0xef, 0x00, 0xff]),
+  },
+  {
+    video: 'red-green.bt601.vp9.webm',
+    redExpectation: new Uint8Array([0xd9, 0x00, 0x00, 0xff]),
+    greenExpectation: new Uint8Array([0x01, 0xef, 0x00, 0xff]),
+  },
+  {
+    video: 'red-green.bt709.vp9.webm',
+    redExpectation: new Uint8Array([0xff, 0x00, 0x00, 0xff]),
+    greenExpectation: new Uint8Array([0x00, 0xff, 0x00, 0xff]),
+  },
+  {
+    video: 'red-green.bt2020.vp9.webm',
+    redExpectation: new Uint8Array([0xff, 0x00, 0x00, 0xff]),
+    greenExpectation: new Uint8Array([0x00, 0xff, 0x00, 0xff]),
+  },
 ];
```

Comment:
If you use `as const` here you probably won't need to define the `VideoExpectations` type.
```
] as const;
```